### ISSUE TITLE
ATC 5th - add synonym

### ISSUE THREAD

**Issue by vojtechhuser** [OHDSI/Vocabulary-v5.0#189]
an isolated atc5 name is hard to interpret.

having some mechanism (e.g., path to it as synonym) to see that would be good.

**Comment by dimshitc** [comment]
I like this idea. 
@cgreich , @aostropolets ,what do you think?

**Comment by aostropolets** [comment]
Yep. Vojtech created this issue not to forget about our decision on one of the ATC WG meetings

**Comment by cgreich** [comment]
Agreed. We will add the additional invisible features from the WHO site.

**Comment by dimshitc** [comment]
like what?
for examlple for B01AC Platelet aggregation inhibitors excl. heparin
https://www.whocc.no/atc_ddd_index/?code=B01AC06&showdescription=yes
all this explanation?

**Comment by cgreich** [comment]
Your examples touches the indication. That's not easy, because of the long paragraph. But the rest we could. In this case: "acetylsalicylic acid, 1 tablet daily, oral administration, independent of strength"

**Comment by cgreich** [comment]
We may even swap concept_name and synonym_name.
